pkg/backup/client.go

```diff
@@ -377,6 +380,7 @@ func (bc *Client) backupRange(
 
 	bc.backupMeta.StartVersion = backupTS
 	bc.backupMeta.EndVersion = backupTS
+	bc.backupMeta.IsRawKv = isRawKv
```

Comment:
I've updated kvproto and added backed up range in backupMeta. Maybe you need also set the range then.

Reply:
[Done](https://github.com/pingcap/br/pull/101/files#diff-7dfd05fbd1b9304c3d90dc85e3ad8090R388)